src/models/user.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+import mongoose, { Schema } from 'mongoose';
+
+export const STATUS_ACTIF = 'actif';
+export const STATUS_INACTIF = 'inactif';
+export const CUSTOMER_ROLE = 'consumer';
+export const COURSIER_ROLE = 'rider';
+
+const userSchema = new Schema({
+  services: {
+    facebook: String
+  },
+  role: {
+    type: String,
+    enum: [CUSTOMER_ROLE, COURSIER_ROLE],
+    default: CUSTOMER_ROLE
+  },
+  location: {
+    type: {
+      type: String,
+      enum: ['Point'],
+      default: 'Point'
+    },
+    coordinates: {
+      type: [Number],
+      default: [34.02590720000001,-6.836439899999959]
```

Comment:
because the location is index  propriety, location is required for create an user,  and in first time i dont know the location of user because i receive the data from facebook.